pages/signin/index.tsx

```diff
@@ -1,13 +1,86 @@
 import { Button } from '../../components/buttons';
+import { Card, CardContent, Typography, Container } from '@material-ui/core';
 import { Layout } from '../../containers/layout';
+import InputField from '../../components/input-field/InputField';
+import { useState } from 'react';
+import { PageProps, UserInput } from '../../lib/types';
+import { withTranslation } from '../../i18n';
+import styles from './Signin.module.css';
+import { useAuth } from '../../hooks/useAuth';
+import { FirebaseUser } from '../../lib/types/auth';
+
+const SignInPage = ({ t }: PageProps): React.ReactElement => {
+  const [formValues, setFormValues] = useState<Partial<UserInput>>({
+    name: {
+      lastName: '',
+    },
+    gender: 'OTHER',
+    email: '',
+    password: '',
+    type: 'TALENT',
+  });
+  const auth = useAuth();
+  console.log(auth.user);
+
+  const handleLogout = (e: React.MouseEvent<HTMLButtonElement>): void => {
+    e.preventDefault();
+    auth.logout();
+  };
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>): void => {
+    e.preventDefault();
+    if (formValues.email && formValues.password) {
+      auth
+        .signin(formValues.email, formValues.password)
+        .then((user: FirebaseUser | void | null) => {
+          if (typeof user !== null) {
+            console.log('current user: ', auth.user);
+          } else {
+            console.log(
+              'wrong email or password, current user still is: ',
```

Comment:
in the end, we will inform the user that username or password is wrong, and do something about forgotten passwords, right? Maybe add a TODO?

Reply:
Yes, this is part of the error handling TODO.
Added this to the Jira ticket - just for our reference.

Reply:
You already created the ticket, thanks!